core/src/services/onedrive/backend.rs

```diff
@@ -16,111 +16,54 @@
 // under the License.
 
 use std::fmt::Debug;
+use std::fmt::Formatter;
 use std::sync::Arc;
 
-use bytes::Buf;
-use bytes::Bytes;
-use http::header;
-use http::Request;
 use http::Response;
 use http::StatusCode;
 
-use super::delete::OnedriveDeleter;
+use super::core::OneDriveCore;
+use super::delete::OneDriveDeleter;
 use super::error::parse_error;
-use super::graph_model::CreateDirPayload;
-use super::graph_model::ItemType;
-use super::graph_model::OneDriveUploadSessionCreationRequestBody;
-use super::graph_model::OnedriveGetItemBody;
-use super::lister::OnedriveLister;
+use super::lister::OneDriveLister;
 use super::writer::OneDriveWriter;
 use crate::raw::*;
 use crate::*;
 
 #[derive(Clone)]
-pub struct OnedriveBackend {
-    info: Arc<AccessorInfo>,
-    root: String,
-    access_token: String,
-    client: HttpClient,
+pub struct OneDriveBackend {
```

Comment:
The naming here needs to align with the scheme. Since our scheme is `onedrive` instead of `one_drive`, we must use `Onedrive` instead of `OneDrivew`.

Reply:
Ooops, good catch. I didn't mean to change this struct name. Will push a fix.